src/officehours_api/notifications.py

```diff
@@ -12,10 +12,18 @@
 
 from officehours_api.models import Queue, Meeting, MeetingStatus
 
-logger = logging.getLogger(__name__)
-
-twilio = TwilioClient(settings.TWILIO_ACCOUNT_SID, settings.TWILIO_AUTH_TOKEN)
-
+def initialize_twilio():
+    logger = logging.getLogger(__name__)
+
+    if (settings.TWILIO_ACCOUNT_SID and settings.TWILIO_AUTH_TOKEN and settings.TWILIO_MESSAGING_SERVICE_SID):
+        twilio_client = TwilioClient(settings.TWILIO_ACCOUNT_SID, settings.TWILIO_AUTH_TOKEN)
+        logger.info("Twilio client initialized.")
+    else:
+        logger.info("Twilio client setup skipped. Twilio settings values are not set (TWILIO_ACCOUNT_SID, TWILIO_AUTH_TOKEN, TWILIO_MESSAGING_SERVICE_SID).")
```

Comment:
Should the logging level be "warn" instead of "info" here?

Reply:
Not sure if we're using the logger warning elsewhere in the app, but I don't think it'd impact user experience, so went ahead with using a warning.